src/dateTime.js

```diff
@@ -3,17 +3,23 @@ function getDateTime() {
   var month = parseInt(dateObj.getMonth() + 1);
   var day = parseInt(String(dateObj.getDate() + 1).padStart(2, '0'));
   var year = parseInt(dateObj.getFullYear());
+  // hour is like this coz getHours() is a bit whack, shows 2300 when it is 11am and 1100 when it is 11pm
   var hour = (parseInt(dateObj.getHours()) + 1 > 12) 
               ? parseInt(dateObj.getHours()) + 1 - 12 
               : parseInt(dateObj.getHours()) + 1 + 12;
   var min  = parseInt(dateObj.getMinutes());
+
+  var ewDate = day + '/' + month  + '/'+ year + 'Ew';
```

Comment:
Let's remove + "Ew" , lines 12 and 13. 
Add it in makeRequest when appending (Make this change in your latest pr)